packages/component-library/src/components/table-and-list/mt-data-table/mt-data-table.vue

```diff
@@ -9,40 +9,48 @@
           :model-value="searchValue"
           @change="emitSearchValueChange"
         />
+        <template v-if="filters.length > 0">
```

Comment:
@Haberkamp I think this implementation could be unstable very quickly. You are more into the topic of filters. Do you have a better, more defined approach for adding custom filter types?

I could imagine a general slot which allows adding custom "filter renderer"

Reply:
FYI, the current behavior is not proper with tablet, because it's open a popover, we want to customize the filter button, it will open the modal instead.